### ISSUE TITLE
update zapper api to check balances to v2

### ISSUE THREAD

**Issue by Sednaoui** [Sednaoui/Candide#76]

**Comment by CherryDT** [comment]
Note: Public API is no longer available and this key is no longer working. People need to request a private API key now.

**Comment by Sednaoui** [comment]
> Note: Public API is no longer available and this key is no longer working. People need to request a private API key now.

@CherryDT you're right. I just noticed that the public APIs aren't available anymore. We do have a private API internally that we will be using in production, we will use it one we migrate to zapper v2.
